vidpub/__main__.py

```diff
@@ -41,6 +32,16 @@
 TIMEZONE_TAIPEI = pytz.timezone("Asia/Taipei")
 
 
+# Video data generator variables
+YOUTUBE_API_KEY = os.environ.get("YOUTUBE_API_KEY", "")
+CHANNEL_ID = os.environ.get("CHANNEL_ID", "")
+PLAYLIST_TITLE = os.environ.get("PLAYLIST_TITLE", "")
+PLAYLIST_ID = os.environ.get("PLAYLIST_ID", "")
+CONFERENCE_DAY1 = os.environ.get("CONFERENCE_DAY1", "")
+
+MAX_RESULT_LIMIT = 100
+
```

Comment:
These variables could be put in `generate_playlist` function if they are only used in there.

Reply:
Done!